android/proguard-mwm.txt

```diff
@@ -1,38 +1,9 @@
-# obfuscate supportV7 menu subclasses. it fixes bug with some Samsung and other devices ROMS based on android 4.2.2.
-# more details here : https://code.google.com/p/android/issues/detail?id=78377
-# For some reason, this line disables optimizations and avoids crashes due to missing @Keep attributes.
-# Looks like R8 keeps EVERYTHING except that unused support lib :)
-# TODO: Remove this line after properly marking all JNI-called classes and methods with @Keep.
-# Also remove everything else what is not needed.
--keep class !android.support.v7.internal.view.menu.**,** {*;}
+# Every JNI-called class and method should have a @Keep annotation to properly work and to avoid
```

Comment:
What is the purpose of this patch?

Reply:
To enable Java code optimizations and shrinking.